python/server/ServerWrapper.cpp

```diff
@@ -116,7 +119,7 @@ namespace {
         } else {
             Empire* empire = GetEmpire(empire_id);
             if (!empire) {
-                ErrorLogger() << "PythonServerWrapper::GenerateSitRep: couldn't get empire with ID " << empire_id;
+                ErrorLogger() << "ServerWrapper::GenerateSitRep: couldn't get empire with ID " << empire_id;
```

Comment:
__FILE__ and __LINE__ would work fine. Could be put into the macro for ErrorLogger() or put in case-by-case when wanted.